qutebrowser/javascript/pac_utils.js

```diff
@@ -0,0 +1,215 @@
+// This is based on nsProxyAutoConfig.js from Mozilla.
```

Comment:
This should probably have an appropriate copyright notice so we don't run into trouble with distributions packaging qutebrowser.

Which version of this file from where is this based from? Can you include a link to the original file, plus the copyright/license header (if any) from there?